app/scripts/components/common/map/style-generators/raster-paint-layer.tsx

```diff
@@ -43,11 +43,10 @@ export function RasterPaintLayer(props: RasterPaintLayerProps) {
 
   useEffect(
     () => {
-      if (!assetUrl) return;
-
+      // in the case of titiler-cmr, there is no asset url. The asset urls are determined internally by titiler-cmr.
       const tileParams = qs.stringify({
-        url: assetUrl,
-        time_slice: date,
+        ...(assetUrl && { url: assetUrl }), // Only include `url` if `assetUrl` is truthy (not null or undefined)
```

Comment:
🤔  I wonder if it is best if the raster-paint-layer doesn't need to care about the specifics of the layer- How about we move this logic to Zarr-time-series? 
I picture something like the below in zarr-timeseries layer
```
  const zarrSourceParams = { ...sourceParams, assetUrl};
 return <RasterPaintLayer {...props} sourceParams={zarrSourceParams} />;
```

Reply:
yes I had it that way at one point actually so happy to make this change.